atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLock.java

```diff
@@ -245,6 +359,10 @@ private CASResult writeLockWithCas(
         );
     }
 
+    private Column lockColumnWithStrValue(String strValue) {
```

Comment:
There's already a `lockColumnWithValue` that accepts a `byte[]` input. This is basically a wrapper around that.